### PR TITLE
[14.0][MIG] shopinvader_partner_firstname: Migration to 14.0

### PR DESCRIPTION
**Pull request by ivantodorovich**

### CODE REVIEW

**Review by jcoux — APPROVED** [review]

File: shopinvader_partner_firstname/__manifest__.py [hunk 1]
```diff
@@ -1,16 +1,13 @@
-# Copyright 2019 Camptocamp SA
+# Copyright 2021 Camptocamp SA
```
Comment (simahawk) [inline shopinvader_partner_firstname/__manifest__.py]: rollback this please: copyright should always state the 1st here

File: shopinvader_partner_firstname/tests/test_firstname.py [hunk 2]
```diff
@@ -1,25 +1,57 @@
 # Copyright 2017 Akretion (http://www.akretion.com).
```
Comment (simahawk) [inline shopinvader_partner_firstname/tests/test_firstname.py]: I think this old copyright is wrong as I probably copy/pasted the header: I've created the module in 2019. 
Please set the year to 2019 and remove Akretion.

File: shopinvader_partner_firstname/tests/test_firstname.py [hunk 3]
```diff
@@ -1,25 +1,57 @@
...
+        params.pop("lastname")
+        partner_id = self.service.dispatch("create", params=params)["data"]["id"]
+        partner = self.env["res.partner"].browse(partner_id)
+        self._test_partner_data(partner, self.data)
+
+    def test_create_customer_exclude(self):
+        """Test that we can't use name with firstname and lastname"""
+        # Case 1: name, firstname and lastname can't all be together
+        params = self.data.copy()
+        with self.assertRaises(UserError):
```
Comment (simahawk) [inline shopinvader_partner_firstname/tests/test_firstname.py]: as mentioned in #1002 please check that the exception is the one you expect

File: shopinvader_partner_firstname/services/address.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Camptocamp (http://www.camptocamp.com)
```
Comment (simahawk) [inline shopinvader_partner_firstname/services/address.py]: 2019

**Review by simahawk — CHANGES_REQUESTED** [review]

File: shopinvader_partner_firstname/__manifest__.py (hunk 1 above)
Comment (ivantodorovich) [reply shopinvader_partner_firstname/__manifest__.py]: got it :)

**Review by simahawk — APPROVED** [review]

**Pull request merged**